src/components/LikePage/LikePage.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+import AttractionCard from '../AttractionCard/AttractionCard'
+import Searchbar from '../Searchbar/Searchbar'
+
+export default function LikePage({
+  allSights,
+  likedPlaces,
+  onLikePlace,
+  userInput,
+  setUserInput,
+}) {
+  const filteredPlaces = allSights.filter(sights =>
+    likedPlaces.includes(sights.name)
+  )
+  return (
+    <PageLayout>
+      <Searchbar userInput={userInput} setUserInput={setUserInput} />
+      <Heading>Your liked Places</Heading>
+      {likedPlaces.length === 0 && <p>You haven't liked anything yet!</p>}
+      {filteredPlaces
+        .filter(item =>
+          item.name.toLowerCase().includes(userInput.toLowerCase().trim())
+        )
```

Comment:
Maybe you can move this into the `filteredPlaces` declaration in line 12 so that you don't have to filter your `filteredPlaces`again.

Reply:
I also think this would be a good improvement :)

Reply:
Great idea! Thank you